clients/include/testing_iamax_iamin_strided_batched.hpp

```diff
@@ -52,8 +52,8 @@ hipblasStatus_t testing_iamax_iamin_strided_batched(const Arguments&
 
         if(batch_count > 0)
         {
-            host_vector<int> cpu_0(1);
-            host_vector<int> gpu_0(1);
+            host_vector<int> cpu_0(batch_count);
+            host_vector<int> gpu_0(batch_count);
             CHECK_HIP_ERROR(hipMemcpy(
                 gpu_0, d_hipblas_result_0, sizeof(int) * batch_count, hipMemcpyDeviceToHost));
             unit_check_general<int>(1, batch_count, 1, cpu_0, gpu_0);
```

Comment:
Can't select the line 62, but why is success returned and not the func status ?

Reply:
We could, but it shouldn't make a difference at this point since we have the CHECK_HIPBLAS_ERROR. I think the status return in hipBLAS is kind of moot now since we have that macro, could change it to return void like rocBLAS at some point I think.

Reply:
Okay, so all the expected values are internal now.  I'll have to double check rocblas,  I guess the return should be a gtest expected value, not sure what they support again or how the test harness is used so will have to review that first. thanks